views/home.erb

```diff
@@ -5,6 +5,15 @@
 <h1 class="dcf-txt-h3">Welcome to Innovation Studio Manager</h1>
 
 <h5>Hello, <%= @user.full_name %> (<%= @user.username %>)</h5>
+<% if @user.expiration_date > Date.today%>
+    <% if @user.expiration_date > (Time.now + (2*7*24*60*60)) %>
+        <h2 > Your Membership will expire on: <%= @user.expiration_date.to_formatted_s(:long_ordinal)[0...-5] %></h2>
+    <% else %>
+        <h2 style="color:yellow;"> Your Membership will expire on: <%= @user.expiration_date.to_formatted_s(:long_ordinal)[0...-5] %> </h2>
+    <% end %>
+<% else%>
```

Comment:
Again very minor :) can we get a space between else and %>

Reply:
Done.